### PR TITLE
Metabot documentation

### LINKED ISSUE #2196: documentation about Metabot

### PR DESCRIPTION
**Pull request by kdoh**
closes #2196

### CODE REVIEW

File: docs/users-guide/10-metabot.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Getting answers in Slack with Metabot
+
+You can already send data to Slack on  a set schedule via [Pulses](http://www.metabase.com/docs/latest/users-guide/07-pulses) but what about when you need an answer right now?. Say hello to Metabot.
+
+Metabot helps add context to conversations you’re having in Slack by letting you see results from Metabase as part of the conversation.
+
+## Connecting to Slack.
+To use Metabot with slack you’ll first need to connect Metabase to your Slack with an API token.
```
Comment (agilliland) [inline docs/users-guide/10-metabot.md]: capitalize `Slack`

File: docs/users-guide/10-metabot.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Getting answers in Slack with Metabot
+
+You can already send data to Slack on  a set schedule via [Pulses](http://www.metabase.com/docs/latest/users-guide/07-pulses) but what about when you need an answer right now?. Say hello to Metabot.
```
Comment (mazameli) [inline docs/users-guide/10-metabot.md]: > an answer right now?. Say

There's an errant period after this question mark.

File: docs/users-guide/10-metabot.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Metabot helps add context to conversations you’re having in Slack by letting you see results from Metabase as part of the conversation.
```
Comment (mazameli) [inline docs/users-guide/10-metabot.md]: Consider changing to, "Metabot helps add context to conversations you’re having in Slack by letting you insert results from Metabase."

File: docs/users-guide/10-metabot.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+Metabot can show individual questions and also lists of questions that have already been asked in Metabase.
+
+If you ever need help remembering what Metabot can do, just type ```metabot help``` in Slack.
+
+![Metabot help](images/metabot/MetabotHelp.png)
+
+## Showing questions
+
+To see a question from Metabase in Slack type
+“metabot show” and then specify the name of a saved question. If you have several similarly named questions Metabot will ask you to differentiate between the two by typing the number next to the name.
```
Comment (mazameli) [inline docs/users-guide/10-metabot.md]: Don't you have to write the name of the question with quotes around it, "like this?"

File: docs/users-guide/10-metabot.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+![Metabot similar](images/metabot/MetabotSimilarItems.png)
+
+That number is the ID number of the question in Metabase and if you find yourself using the same question over and over again you can save a bit of time by typing “metabot show 19”
```
Comment (mazameli) [inline docs/users-guide/10-metabot.md]: > "metabot show 19"

missing a period

File: docs/users-guide/10-metabot.md (hunk 4 above)
Comment (kdoh) [inline docs/users-guide/10-metabot.md]: Yeah if you don't put quotes around the string metabot will basically just grab the first word and go off of that. So it's not "required" necessarily but it does change the results quite a bit.

**Pull request merged**